### ISSUE TITLE
Trying out a thing!

### ISSUE THREAD

**Issue by Alexs1214** [kaseys111/dwralums#1]
Hey does this work as a means of suggesting ideas for the site?

**Comment by kaseys111** [comment]
Yeah, but can you not use the Projects tab instead? Just saw this, sorry....

**Comment by Alexs1214** [comment]
I cant add anything to the projects tab....or at least I dont see how I
can. I think maybe thats just for contributors?
